### PR TITLE
fmt: add fmt nightly with `imports_granularity` crate

### PR DESCRIPTION
**Pull request by tcoratger**
Time spent on this PR:

Resolves: #<Issue number>

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?

# Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: crates/sequencer/Cargo.toml [hunk 1]
```diff
@@ -76,7 +76,7 @@ use_self = "allow"
 name = "sequencer"
 version = "0.1.0"
 edition = "2021"
-rust-version = "1.76"
+rust-version = "1.79"
```
Comment (tcoratger) [inline crates/sequencer/Cargo.toml:79]: Seems like it's a better practice to put the rust version here. I observed that the rust-toolchain file overrides almost everything. And seems like a good place to put this

https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

Comment (tcoratger) [reply crates/sequencer/Cargo.toml:79]: Oups my comment is on the sequencer crate but I wanted to comment this also for our high level cargo file

Comment (tcoratger) [reply crates/sequencer/Cargo.toml:79]: Here it's the minimal version required, meaning that the user can only compiler with higher version of rust:

> The rust-version field is an optional key that tells cargo what version of the Rust language and compiler your package can be compiled with. If the currently selected version of the Rust compiler is older than the stated version, cargo will exit with an error, telling the user what version is required.

**Comment by tcoratger** [comment]
@greged93 Let me know when we can merge so that I can handle the conflicts just before we merge as this pr is subjected to a lot of conflicts each time we update the repo :)

**Comment by greged93** [comment]
Ah sorry, will merge!

**Review by greged93 — APPROVED** [review]

**Review by greged93 — APPROVED** [review]

**Pull request merged**